package.json

```diff
@@ -94,5 +94,8 @@
     "sinon": "^1.17.4",
     "sinon-as-promised": "^4.0.0",
     "sinon-chai": "^2.8.0"
+  },
+  "optionalDependencies": {
```

Comment:
I have no practical experience with the feature. Per the documentation, npm will always try to install optionalDependencies with `npm install` but the overall install process will not fail of the optionalDependency fails to install. This seems like the right construct to include a module that requires a higher minimum version of node than the CLI's minimum.